geographic-location.md

```diff
@@ -0,0 +1,23 @@
+Anbox Cloud allows specifying a static geographic location which is provided to the Android container.  The location is configured in the file `/var/lib/anbox/static_gps_position` within the Anbox container.
```

Comment:
@adglkh didn't you add an API in Anbox to allow changing that dynamically ?

Reply:
We support specifying static gps data via  `/var/lib/anbox/static_gps_position` from the very beginning of location feature support as described above. Then we add support to dynamically change the location via HTTP API. For the backward compatibility purpose, we need to keep both.  
As @ru-fu created a dedicated page to introduce this feature,  I'd suggest having two sub-headers on this page to make it clear on the way of changing GPS data through:
* static GPS data
* HTTP API

Regarding the GPS data change through HTTP API, we could navigate people to the following section https://github.com/anbox-cloud/discourse-docs/blob/main/anbox-http-api.md#heading--10location